var/spack/repos/builtin/packages/git/package.py

```diff
@@ -387,12 +386,21 @@ def build(self, spec, prefix):
             args.append('NO_GETTEXT=1')
         make(*args)
 
+        if spec.satisfies('platform=darwin'):
+            with working_dir('contrib/credential/osxkeychain'):
+                make()
+
+
     def install(self, spec, prefix):
         args = ["install"]
         if '~nls' in self.spec:
             args.append('NO_GETTEXT=1')
         make(*args)
 
+        if spec.satisfies('platform=darwin'):
+            copy('contrib/credential/osxkeychain/git-credential-osxkeychain',
```

Comment:
```suggestion
            install('contrib/credential/osxkeychain/git-credential-osxkeychain',
```
Pretty sure we have a wrapper for this. Not sure if there's much of a difference other than permissions.

Reply:
Thanks for the heads-up, I confirmed that `install` works as expected.

Reply:
Motherfucking style 🙄